app/templates/direct-award/view-project.html

```diff
@@ -24,17 +24,107 @@
 {% endblock %}
 
 {% block main_content %}
-
-<div class="index-page">
-  <header class="page-heading">
-    <h1>
-      Project - {{ project_name }}
-    </h1>
-  </header>
   <div class="grid-row">
-    <p>{{ search_summary }}</p>
-    <p>{{ search_created_at|utcdatetimeformat }}</p>
-    <a href="{{ search_page_url }}">View your search</a>
+    <div class="column-two-thirds">
+      {% with
+        heading = project.name|capitalize_first
```

Comment:
Yeah, I think if we really want this then it'd be better done in one place somehow, anyway.